jenkins-integration/jenkins-jobs/common/scripts/job-steps/035_customize_hocon_settings.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+pushd jenkins-integration
+source jenkins-jobs/common/scripts/job-steps/initialize_ruby_env.sh
+
+set -x
+set -e
+
+# Setup SSH agent for SSH access to SUT
+eval $(ssh-agent -t 24h -s)
+ssh-add ${HOME}/.ssh/id_rsa
+
+bundle exec beaker \
+        --config hosts.yaml \
+        --load-path lib \
+        --log-level debug \
+        --no-color \
+        --tests \
+beaker/install/shared/35_customize_hocon_settings.rb
+#beaker/install/shared/99_restart_server.rb
```

Comment:
Looks like there's 5 or 6 steps that each restart the server, which seems kind of extreme. Maybe a rollup-restart pipeline step would be good